<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_ShareCourseware</class>
 <widget class="QWidget" name="DB_ShareCourseware">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_title">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_title{
border: none;
background-color: rgb(78,155,252);
border-top-left-radius: 8px;
border-top-right-radius: 8px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>391</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>分享课件</string>
    </property>
   </widget>
   <widget class="DB_CustomLabel_1" name="label_close">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(&quot;:/images/courseware/close-3.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>16</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(':/images/file-1.png');</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 85, 85);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(135, 135, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>367</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_2{
border: none;
background-color: rgba(225,225,225);
border-radius: 8px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_linkName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 85, 85);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_viewLink">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 85, 85);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(135, 135, 135);</string>
   </property>
   <property name="text">
    <string>分享设置：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_private">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>55</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
	color: rgb(135, 135, 135);
}

QRadioButton::indicator:unchecked {
 	border-image: url(&quot;:/images/courseware/Unselected.png&quot;);
}

QRadioButton::indicator:checked {
 	border-image: url(&quot;:/images/courseware/Select.png&quot;);
}</string>
   </property>
   <property name="text">
    <string>加密</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_public">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>55</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
	color: rgb(135, 135, 135);
}

QRadioButton::indicator:unchecked {
 	border-image: url(&quot;:/images/courseware/Unselected.png&quot;);
}

QRadioButton::indicator:checked {
 	border-image: url(&quot;:/images/courseware/Select.png&quot;);
}</string>
   </property>
   <property name="text">
    <string>公开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(135, 135, 135);</string>
   </property>
   <property name="text">
    <string>有效期：</string>
   </property>
  </widget>
  <widget class="DB_CustomComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>491</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:  rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pShare">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>92</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:  rgb(78,155,252);
border: none;
border-radius: 5px;
color: white;
}</string>
   </property>
   <property name="text">
    <string>分享链接</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_line1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>491</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:  rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DB_CustomLabel_1</class>
   <extends>QLabel</extends>
   <header location="global">CustomWidget/db_customlabel.h</header>
  </customwidget>
  <customwidget>
   <class>DB_CustomComboBox</class>
   <extends>QFrame</extends>
   <header location="global">CustomWidget/db_customcombobox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
